src/bin/collapse-dtrace.rs

```diff
@@ -18,6 +18,15 @@ use inferno::collapse::Collapse;
     "
 )]
 struct Opt {
+    /// Include offsets
+    #[structopt(long = "includeoffset")]
```

Comment:
There's a lot of this kind of alphabetization and code-movement for `use` statements that makes this PR a little hard to review. I know it's a bunch of work, but any chance you could split those changes into their own PR so I can review the multicore changes specifically in isolation?

Reply:
👍